libsolidity/formal/BMC.cpp

```diff
@@ -424,6 +535,28 @@ bool BMC::visit(TryStatement const& _tryStatement)
 	return false;
 }
 
+bool BMC::visit(Break const& breakStatement)
```

Comment:
```suggestion
bool BMC::visit(Break const&)
```
Then you can remove the `(void)...` line.